ftw/showroom/js/src/showroom.js

```diff
@@ -146,21 +147,28 @@ module.exports = function Showroom(items = [], options) {
     items = items.map(item => Item(item));
     items.map(item => $(item.element).on("click", select));
     register.append(items);
+    checkArrows();
   }
 
   function prepend(nodes) {
     items = Array.prototype.slice.call(nodes);
     items = items.map(item => Item(item));
     items.map(item => $(item.element).on("click", select));
+    if (options.offset > 0) {
+      options.offset -= items.length;
+    }
     register.prepend(items);
+    checkArrows();
   }
 
-  function reset(items = []) {
+  function reset(items = [], offset = 0) {
```

Comment:
mhh i change the code in the meantime to call `setOffset` with the passed-in offset, so i'd suggest to change the code when we need to reset more stuff, (if you agree).